### PR TITLE
Allow to parse legacy invalid images for ps/status commands

### PR DESCRIPTION
**Pull request by shagren**
refs #971

### CODE REVIEW

File: neuromation/api/jobs.py [hunk 1]
```diff
@@ -361,8 +361,13 @@ def _http_port_from_api(data: Dict[str, Any]) -> HTTPPort:
 
 
 def _container_from_api(data: Dict[str, Any], parser: _ImageNameParser) -> Container:
+    try:
+        image = parser.parse_remote(data["image"])
+    except ValueError:
+        image = RemoteImage(name=f'Invalid Image Name {data["image"]}')
```
Comment (asvetlov) [inline neuromation/api/jobs.py]: I'm curious how this super-long-line looks in tables view provided by `TabularJobsFormatter`?

Comment (shagren) [reply neuromation/api/jobs.py]: It is why I propose to add new Class, then we can use different formatting for `ps` and `status`.
I.e. `InvalidImage` for `ps` and `Invalid Image Name [Name here]` for `ps`

Comment (asvetlov) [reply neuromation/api/jobs.py]: Do you really want to print this invalid image name?
Now you cannot create such a thing, the only way to get invalid image is retrieving historical old enough job. Missing exact invalid image name doesn't make much harm.
I don't insist though. If you prefer your version -- please go ahead and merge.

The very minor and rare thing anyway.

Comment (shagren) [reply neuromation/api/jobs.py]: Agree, will change

**Pull request merged**